lib/base/models/enums/enums_plan/absolute_direction.dart

```diff
@@ -31,16 +32,29 @@ extension CompassDirectionExtension on AbsoluteDirection {
   };
 
   static const icons = <AbsoluteDirection, IconData>{
-    AbsoluteDirection.north: Icons.arrow_upward,
-    AbsoluteDirection.northeast: Icons.arrow_upward,
-    AbsoluteDirection.east: Icons.arrow_forward,
-    AbsoluteDirection.southeast: Icons.arrow_downward,
-    AbsoluteDirection.south: Icons.arrow_downward,
-    AbsoluteDirection.southwest: Icons.arrow_downward,
-    AbsoluteDirection.west: Icons.arrow_back,
-    AbsoluteDirection.northwest: Icons.arrow_upward,
+    AbsoluteDirection.north: Icons.north,
+    AbsoluteDirection.northeast: Icons.north_east,
+    AbsoluteDirection.east: Icons.east,
+    AbsoluteDirection.southeast: Icons.south_east,
+    AbsoluteDirection.south: Icons.south,
+    AbsoluteDirection.southwest: Icons.south_west,
+    AbsoluteDirection.west: Icons.west,
+    AbsoluteDirection.northwest: Icons.north_west,
```

Comment:
For the start instruction, a single arrow up is sufficient, no arrows for absolute direction